packages/driver/src/conUtils.ts

```diff
@@ -137,6 +136,10 @@ export type ResolvedConnectConfigReadonly = Readonly<
   >
 >;
 
+function getEnv(envName: string, _required = false): string | undefined {
+  return process.env[envName];
```

Comment:
Do we need to handle the Deno specific stuff here that was in adaptor.shared.deno.ts?

Reply:
nope! Deno supports the `process` global: https://docs.deno.com/runtime/fundamentals/node/#node.js-global-objects